README.md

```diff
@@ -14,7 +14,6 @@
       - [Manually](#manually)
     - [Build and run the service locally](#build-and-run-the-service-locally)
   - [Useful commands](#useful-commands)
-    - [DB migration](#db-migration)
```

Comment:
This section is already removed from the README, so the anchor doesn't work anymore.